src/bufr_interface.h

```diff
@@ -333,7 +333,7 @@ extern "C" {
 /**
  * Get the bufr node idx for the start node of the subset.
  *
- * @param lun - pointer for the file stream.
+ * @param lun - File ID.
```

Comment:
Do 'bufr_unit' and 'lun' basically mean the same thing?

Reply:
bufr_unit is typically the Fortran logical unit number on the system that's associated with a file, whereas lun is an internal (to the library) index used to keep track of all such logical unit numbers that have already been introduced to the library via calls to openbf.  So they are different entities, and we agreed in a previous discussion to refer to lun as "File ID" going forward, instead of some of the other descriptions (e.g. I/O stream index, etc.) that we've used in the past.

We're trying to update all of these as we go forward, but as you can see there are still some left to go, and I see now that the bufr_file variable name for nemtab and nemtbb in this file should really be lun, so I'll get that cleaned up.